.github/actions/build_image/action.yml

```
@@ -121,9 +124,13 @@ runs:
       id: image_version_branch
       run: |
         base_image=$(echo ${{ inputs.image_name }} | cut -d '-' -f1)
-        app_version=$(podman run --pull=never pulp/${{ inputs.image_name }}:ci-amd64 bash -c "pip3 show pulpcore | sed -n -e 's/Version: //p'")
+        if [[ "${{ env.BUILD }}" == "true" ]]; then
```

Should we not rely on the env here?
```suggestion
        if [[ "${{ steps.rebuild_needed.outputs.build }}" == "true" ]]; then
```

---

Yep, remove `env.BUILD` since it's no longer needed.